server/test/handlers/getAppointments.test.js

```diff
@@ -0,0 +1,80 @@
+const appointmentHandler = require('../../models/handlers/appointment');
```

Comment:
Hi Amy, 
I tried to run the automated tests in the server folder  with the command npm test. I face this error (see the screenshot). Can you suggest a reason for this error?
<img width="696" alt="Screen Shot 2020-10-30 at 10 31 57 AM" src="https://user-images.githubusercontent.com/71143782/97717653-6d3e8900-1a9b-11eb-881a-832a59a0bd9b.png">

Reply:
Hi Nancy,

I would recommend reviewing and following Step 4 in the Local Setup section in the README.md file.